DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -17,6 +17,20 @@ globalscriptsdir=${HCPPIPEDIR_Global}
 eddydir=${workingdir}/eddy
 datadir=${workingdir}/data
 
+echo "Generating eddy QC report in ${workingdir}/qc"
+rm -r ${workingdir}/qc
+qc_command="${FSLDIR}/bin/eddy_quad "
+qc_command+="${eddydir}/eddy_unwarped_images "
+qc_command+="-idx ${eddydir}/index.txt "
+qc_command+="-par ${eddydir}/acqparams.txt "
+qc_command+="-m ${eddydir}/nodif_brain_mask.nii.gz "
```

Comment:
I know none of the rest of the script is whitespace-safe, but would you mind converting this command-building code to the whitespace-safe array pattern anyway?  Note that there needs to be an unquoted space between separate arguments in the same set of parenthesis, `(-idx "${eddydir}/index.txt")`, but separate sets of parenthesis don't require any added space, as the separation is accomplished by being separate array elements, not by literal whitespace.

```
qc_command=("${FSLDIR}/bin/eddy_quad")
qc_command+=("${eddydir}/eddy_unwarped_images")
qc_command+=(-idx "${eddydir}/index.txt")
...
"${qc_command[@]}"
```